website/docs/feature-flag-tutorials/use-cases/user-management-access-controls.md

```diff
@@ -0,0 +1,172 @@
+
+In the Unleash Admin UI, go to **Admin > Roles** to view, create, and manage user roles.
+
+![Manage all user roles in the Unleash Admin UI.](/img/use-case-user-mgmt-root-roles.png)
+
+We have 5 [predefined roles](/reference/rbac#predefined-roles) within our RBAC framework at Unleash.
+
+_Root roles_:
+
+1. Admin
+2. Editor
+3. Viewer
+
+_Project roles_:
+
+1. Owner
+2. Member
+
+Assign users with root roles to configure entire systems. These users can perform any operation within the Unleash platform and change permissions for other users when they belong to a specific user group.
+
+Assign other users to project roles to have domain-specific control within a specific project maintained in Unleash.
+
+Project permissions are separated from root permissions to make it even more targeted regarding what permissions someone can and cannot have for each piece of Unleash. Assign developers with creation and modification rights and viewers who can observe but not change.
+
+For more fine-tuned access controls, create [custom root roles](/how-to/how-to-create-and-assign-custom-root-roles) and [custom project roles](/how-to/how-to-create-and-assign-custom-project-roles), where you can define the privileges and limitations beyond the predefined roles we have built into Unleash.
+
+### Extend RBAC with a change management workflow
+
+While RBAC allows you to administer Unleash safely, you might need approval processes when changing feature flags or their configuration. When multiple teams are working on complex systems, [change requests](/reference/change-requests) provide a systematic approach to:
+
+-   **Comprehensive review**: Every proposed feature flag modification goes through a review, reducing the likelihood of unintended consequences.
+-   **Audit trail**: Every change is documented, timestamped, and attributed to specific team members, creating a permanent record of system modifications.
+-   **Four-eyes approval workflows**: Multi-stage approval processes ensure critical changes are thoroughly reviewed and approved by at least 2 other people before implementation.
+-   **Compliance requirements**: For regulated industries like finance and healthcare, Change Requests provide the detailed documentation necessary to meet strict compliance standards.
+
+![Change requests are divided between two tabs: open and closed change request lists with relevant metadata listed per request.](/img/use-case-user-mgmt-change-requests.png)
+
+Imagine a large banking application where a development team wants to modify a feature flag controlling a new authentication method. Instead of a developer making an immediate change, the change request workflow might require:
+
+1. Initial proposal submission
+2. Security team review
+3. Compliance officer approval
+4. Final sign-off from technical leadership
+
+![This GIF shows how to quickly make changes to your flag, request the change, approve it, and apply the changes.](/img/use-case-user-mgmt-cr.gif)
+
```

Comment:
I would like to add a link to Mateusz's blog on this topic. https://www.getunleash.io/blog/feature-flag-change-requests-how-to

Something like "For more detail on what enterprise-level change requests should include [read our blog](https://www.getunleash.io/blog/feature-flag-change-requests-how-to)